### PR TITLE
Sensor hdc1000, mcp9808, tsl2561, tsl2591

### PR DESCRIPTION
**Pull request by sdalu**
Added sensor hdc1000 and mcp980

### CODE REVIEW

**Comment by utzig** [comment]
Many of those files don't have a license header. If you wrote them please add the license header, otherwise we need to check the original licenses.

File: os/various/bswap.h [hunk 1]
```diff
@@ -0,0 +1,129 @@
+#ifndef BSWAP_H
```
Comment (utzig) [inline os/various/bswap.h]: This still needs a copyright header!

File: os/various/i2c_helpers.h [hunk 2]
```diff
@@ -0,0 +1,267 @@
+#ifndef I2C_HELPERS_H
```
Comment (utzig) [inline os/various/i2c_helpers.h]: copyright notice!

**Comment by sdalu** [comment]
Done

**Pull request merged**